<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>路由器控制</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0,0">
          <item row="3" column="1">
           <widget class="QSlider" name="slider_delay">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="btn_del_delay">
            <property name="text">
             <string>删除延迟</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="btn_loss">
            <property name="text">
             <string>设置丢包率</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btn_delay">
            <property name="text">
             <string>设置延迟</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="btn_del_loss">
            <property name="text">
             <string>删除丢包</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="combodelay">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="slider_loss">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QComboBox" name="combo_loss">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QPushButton" name="btn_jitter_setting">
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="text">
             <string>抖动设置</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSlider" name="slider_jitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QComboBox" name="combo_jitter">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="btn_user_cmd">
            <property name="text">
             <string>执行自定义命令</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>状态</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QComboBox" name="combo_jitter_delay">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QSlider" name="slider_jitter_delay">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="3">
           <widget class="QComboBox" name="combo_custom_editor">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLineEdit" name="lineEdit_status">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_response"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="score_tableView"/>
       </item>
       <item>
        <widget class="QTableView" name="delay_tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_setting">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_export">
       <property name="text">
        <string>导出结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_run">
       <property name="text">
        <string>评分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear">
       <property name="text">
        <string>清理垃圾</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
